src/tools/ResultsComparer/CommandLineOptions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using CommandLine;
+using CommandLine.Text;
+
+namespace ResultsComparer
+{
+    public class CommandLineOptions
+    {
+        [Option("base", HelpText = "Path to the folder/file with base results.")]
+        public string BasePath { get; set; }
+
+        [Option("diff", HelpText = "Path to the folder/file with diff results.")]
+        public string DiffPath { get; set; }
+
+        [Option("merged", HelpText = "Path to the folder/file with results merged for multiple jobs in the same file.")]
```

Comment:
Is this the output directory? If so, it should be stated in the help string.

Reply:
I was not sure how to call it (naming..)

So when we run the Benchmarks with `--runtimes netcoreapp2.1 netcoreapp2.2` BDN is going to create one json file with the results for both 2.1 and 2.2 inside.

This option allows to compare the perf for such files (results for few jobs are merged into one file)

@jorive I am open to better name suggestions ;p